_data/sites.json

```diff
@@ -7238,6 +7238,16 @@
         ]
     },
 
+    {
+        "name": "OfficeDepot",
+        "url": "https://www.officedepot.com/ccpa/landing.do",
+        "difficulty": "hard",
+        "notes": "Note that while their privacy policy states that California residents may have their information deleted permanently, it does not specify whether they delete other accounts also. Go to the URL and fill out the form with the information pertaining to your account. Open the 'Delete My Information' section and check the box next to 'Delete my consumer data'. Submit the form and wait for their confirmation email to arrive.",
```

Comment:
```suggestion
        "notes": "Note that while their privacy policy states that California residents may have their information deleted permanently, it does not specify whether they delete other accounts also. Go to the URL and fill out the form with the information pertaining to your account. Open the 'Delete My Information' section and check the box next to 'Delete my consumer data'. Submit the form and wait for their confirmation email to arrive. If the confirmation arrives and the account is not deleted, you might have to reach out to the customer support and present their privacy policy to have them act.",
```

Reply:
That's excellent! I've tweaked the wording just a bit, and I think this will work.